src/Makefile.in

```diff
@@ -96,7 +96,7 @@ $(DEPFILE) dep: prog/gpm-root.c
 
 	# create dependencies
 	for DEPS in `echo *.c */*.c`; do \
-	$(CC) -I. -I $(srcdir) -M @CPPFLAGS@ $(CPPFLAGS) $$DEPS | \
+	$(CC) -I. -Iheaders -I $(srcdir) -M @CPPFLAGS@ $(CPPFLAGS) $$DEPS | \
```

Comment:
i think you want `-I$(srcdir)/headers` instead ?  it should be rooted to the source path rather than the build path.